### PR TITLE
Add July 2024 Randomized Format Spotlight

### PR DESCRIPTION
**Pull request by livid-washed**
A CAP Pokemon will always generate in slot 2, and can randomly generate in the rest of the team.

Please do not merge until https://github.com/smogon/pokemon-showdown/pull/10376 is merged.

### CODE REVIEW

File: config/formats.ts [hunk 1]
```diff
@@ -2483,49 +2483,29 @@ export const Formats: FormatList = [
...
+	// Randomized Metas
+	///////////////////////////////////////////////////////////////////
+
+	{
+		section: "Randomized Metas",
+		column: 3,
 	},
 	{
-		name: "[Gen 9] Random Roulette",
+		name: "Random Roulette",
```
Comment (KrisXV) [inline config/formats.ts:2502]: this makes it display as [gen 6] on the client, so let's just keep the gen 9 for now.

Comment (KrisXV) [inline config/formats.ts:2502]:
```suggestion
		name: "[Gen 9] Random Roulette",
```

File: data/random-battles/gen9/teams.ts [hunk 2]
```diff
@@ -1526,7 +1526,7 @@ export class RandomTeams {
 
 		// Low Priority
 		if (
-			(species.id === 'garchomp' && role === 'Fast Support') || (
+			(ability === 'Rough Skin') || (
```
Comment (KrisXV) [inline data/random-battles/gen9/teams.ts:1529]:
```suggestion
		(ability === 'Rough Skin' || (
```

File: config/formats.ts (hunk 1 above)
Comment (ACakeWearingAHat) [reply config/formats.ts]: okeydokey

File: data/random-battles/gen9/teams.ts [hunk 3]
```diff
@@ -1526,7 +1526,7 @@ export class RandomTeams {
 
 		// Low Priority
 		if (
-			(species.id === 'garchomp' && role === 'Fast Support') || (
+		(ability === 'Rough Skin' || (
```
Comment (KrisXV) [inline data/random-battles/gen9/teams.ts:1529]:
```suggestion
		ability === 'Rough Skin' || (
```

Comment (ACakeWearingAHat) [reply data/random-battles/gen9/teams.ts]: thank you

**Pull request merged**